### ISSUE TITLE
Improve help for New-SFInitiator

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/PowerShell#29]
```
PS C:\Program Files\SolidFire> get-help New-SFInitiator -full

...

DESCRIPTION
    Creates multiple new initiator IQNs or World Wide Port Names (WWPNs) and optionally assigns them aliases and attributes.

...

    -Name <string>
        The name of the initiator (IQN or WWPN) to create.
...
    -------------------------- EXAMPLE 1 --------------------------

    PS C:\>New-SFInitiator -Name test

    Creates a new initiator with name "test".
    -------------------------- EXAMPLE 2 --------------------------
```

Maybe it's just me, but I didn't realize "-Name" should be an actual IQN, so I used Example 1... From Description it's half-clear that Name *is* IQN (if iSCSI is used) but that could be clearer. And Example 1 is misleading in two ways:

* It uses a non-standard IQN
* It makes it look like Name is Alias (yeah, I should have read Example 2 as well).

NetApp ONTAP generally [disallows](https://library.netapp.com/ecmdocs/ECMP1189419/html/GUID-42BD71D5-722D-4DBC-8E03-B90D646AAB34.html) non-standard IQNs, so it's a good idea (cough, Data Fabric, cough) to encourage consistency.

I propose that names in Examples be changed to something like `-Name iqn.1991-05.com.microsoft:winclient1` and that `-Name <string>` recommends standard (IQN or WWPN) format (even though it may not be enforced). (We should probably enforce that the same way as ONTAP, if we aren't.)  Example:

```
    -Name <string>
        The name of the initiator (using standard IQN or WWPN names) to create.
```

**Comment by ahaid** [comment]
This has been addressed by changing the description of the Name parameter as you suggested and by changing the Example 1 code to be a proper IQN. The release will be available in v1.5.

**Comment by ahaid** [comment]
This issue was fixed in the 1.5 release on Nov 29, 2017.
